spec/lib/evss/disability_compensation_form/form526_to_lighthouse_transform_spec.rb

```diff
@@ -114,6 +125,18 @@
       expect(result.city).to eq('Portland')
       expect(result.dates).not_to be_nil
     end
+
+    it 'sets military/intl address correctly' do
+      data['form526']['veteran']['changeOfAddress']['militaryPostOfficeTypeCode'] = 'APO'
+      data['form526']['veteran']['changeOfAddress']['militaryStateCode'] = 'AE'
+      data['form526']['veteran']['changeOfAddress']['internationalPostalCode'] = '817'
+
+      result = transformer.transform_change_of_address(data['form526']['veteran'])
+      expect(result.city).to eq('APO')
+      expect(result.state).to eq('AE')
+      expect(result.zip_first_five).to eq('817')
+      expect(result.dates).not_to be_nil
```

Comment:
Do we have unit tests for negative scenarios? Like if the data provided is null or not usable for some reason. And if the priority consideration ends up being a thing, tests for that would be good as wel

Reply:
You can't see it fully in this view, but the negative scenario is basically covered by the test right above it